install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -296,19 +296,27 @@ data:
   kube-proxy-replacement:  {{ .Values.global.kubeProxyReplacement | quote }}
 {{- end}}
 {{- if .Values.global.hostServices }}
+{{- if .Values.global.hostServices.enabled }}
```

Comment:
Did you check that all changed options default to `false` in cilium-agent? Otherwise, users won't be able to disable it.

Reply:
Hmm, good point, quite annoying. We have right now:
```
flags.Bool(option.EnableHostReachableServices, false, ...)
flags.Bool(option.EnableNodePort, false, ...)
flags.Bool(option.EnableHostPort, true, ...)
flags.Bool(option.EnableExternalIPs, defaults.EnableExternalIPs -> EnableExternalIPs = true
flags.Bool(option.EnableAutoProtectNodePortRange, true, ...)
```
So HostPort, ExternalIPs and EnableAutoProtectNodePortRange would need an explicitly to be there.

Thinking more about it, I think HostPort and ExternalIPs we could emit only for partial kube-proxy replacement but otherwise leave them out to avoid confusion for users since the daemon overrides these two anyway when probing.

So EnableAutoProtectNodePortRange we would need to leave as-is.